### DISCUSSION TITLE
Apache Lounge is Down, where can i get New Version of Apache

### DISCUSSION THREAD

**Discussion by Emmanuelasika** [leokhoa/laragon#903] [Q&A]
Hello guy: 
[https://www.apachelounge.com](https://www.apachelounge.com/download/) is down, all their servers are off. i tried downloading apache from their official site but no Conf folder. This is my first time working with Laragon and I swear I am so drained already trying to fix this for hours. I just wanted to use PHP 8.3 which requires to update apache version, why isn't upgrading a feature laragon offers automatically? kind of defeats the whole purpose of being what it is.

**Comment by husnilkhatimi** [comment]
Hi, you can try my repackaged version, hopefully can help with your problem

https://github.com/husnilkhatimi/laragonzo

**Comment by pitommasi** [comment]
windows 11 complains about the security of the installer

**Reply by husnilkhatimi** [reply]
If its windows SmartScreen popup, 
you can click 'More info' and then click 'Run anyway'

**Reply by pitommasi** [reply]
the list of downloadables is only shown partially and there's no scrollbar
![immagine](https://github.com/user-attachments/assets/73383e51-6c57-457d-9b64-4f70b672e557)

**Reply by scrollable-sidebar** [reply]
Also, there's no Apache folder, so Apache doesn't run.
![Screenshot 2024-11-07 161103](https://github.com/user-attachments/assets/2e6e24d1-2eb3-4091-9076-4170c1ad0aa0)
![houg7](https://github.com/user-attachments/assets/4a98a1fc-6b39-409e-ae07-88b264d9d836)

**Comment by kaizercodex** [comment]
same problem here

**Comment by pitommasi** [comment]
now apachelounge is up again!

edit: fixed typo

**Comment by masoudnkh** [comment]
That site not have problem and you can use this site below, instead of apachelounge:
https://httpd.apache.org/download.cgi
